packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -211,10 +211,10 @@ class DateTimePicker extends React.Component {
   }
 
   static getDerivedStateFromProps(nextProps, prevState) {
-    if (nextProps.timeZone !== prevState.timeZone) {
+    if (nextProps.initialTimeZone !== prevState.timeZone) {
```

Comment:
Do we want to do this still?

Reply:
No, I don't think we need `getDerivedStateFromProps` anymore since we're going to ignore it.

Reply:
Removed here ea2630e205a708f19fe9d9c16c9f9cbb247d0434